Enterprise-CMCS/macpro-mdct-mcr
# [MDCT-2431] Add extra warning text to incomplete MLR entities

### Description
Adds a new line of warning text to incomplete MLR entity rows. 

Also, adds tests to `getMlrEntityStatus` and the `EntityRow` components.

### Related ticket(s)


MDCT-

---
### How to test
1. Create a new MLR submission
2. Create a new entity.
3. The entity should be incomplete, and there should be new warning text underneath the entity information.
4. Complete entity.
5. Warning text should disappear.

### Important updates
N/A

---
### Author checklist


---


_convert to a different template: [test → val](?expand=1&template=test-to-val-deployment.md)_ | _[val → prod](?expand=1&template=val-to-prod-deployment.md)_
